src/_locales/en_US/messages.json

```diff
@@ -92,7 +92,7 @@
     "message": "Back"
   },
   "sunsetMessage": {
-    "message": "Firefox Private Network will be discontinued on June 15, 2023."
+    "message": "Firefox Private Network will be discontinued on June 20, 2023."
```

Comment:
Confirmed that June 20 is a valid date. (previously date was a Thursday, now it's a Tuesday)

Also did a quick <kbd>grep</kbd> sanity check to make sure the old date wasn't listed elsewhere in the repo:

```sh
git grep -n "June" | cat

src/_locales/en_US/messages.json:95:    "message": "Firefox Private Network will be discontinued on June 20, 2023."
```

Reply:
Thank you kindly, @pdehaan !